package-lock.json

```diff
@@ -9520,9 +9614,9 @@
       "integrity": "sha512-a4UGQaWPH59mOXUYnAG2ewncQS4i4F43Tv3JoAM+s2VDAmS9NsK8GpDMLrCHPksFT7h3K6TOoUNn2pb7RoXx4g=="
     },
     "yargs": {
-      "version": "14.2.0",
-      "resolved": "https://registry.npmjs.org/yargs/-/yargs-14.2.0.tgz",
-      "integrity": "sha512-/is78VKbKs70bVZH7w4YaZea6xcJWOAwkhbR0CFuZBmYtfTYF0xjGJF43AYd8g2Uii1yJwmS5GR2vBmrc32sbg==",
+      "version": "14.2.2",
```

Comment:
14.2.2 resolves the issue, more info: https://github.com/babel/babel/pull/10733#discussion_r347712415

Reply:
> we should not upgrade to v15 of yargs yet as it drops support for node 6

Yep, that's on my radar.